internal/appland/client.go

```diff
@@ -12,11 +12,31 @@ import (
 	"github.com/applandinc/appland-cli/internal/config"
 )
 
-type Client struct {
+type Client interface {
+	BuildUrl(paths ...interface{}) string
+	Context() *config.Context
+	CreateMapSet(app, org string, scenarios []string) (*CreateMapSetResponse, error)
+	CreateScenario(org string, scenarioData io.Reader) (*ScenarioResponse, error)
+	DeleteAPIKey() error
+	Login(login string, password string) error
+	TestAPIKey(apiKey string) (bool, error)
+	delete(url string, body io.Reader) (*http.Response, error)
+	get(url string, body io.Reader) (*http.Response, error)
+	newAuthRequest(method, url string, body io.Reader) (*http.Request, error)
+	post(url string, body io.Reader) (*http.Response, error)
+}
+
+type clientImpl struct {
+	Client
```

Comment:
In golang, an interface is implemented by a struct implicitly just by defining the interface's methods

Reply:
👍 I'll take this out.